src/api/server/common.rst

```diff
@@ -728,6 +734,10 @@
     :json number doc_write_failures: Number of document write failures
     :json number docs_read:  Number of documents read
     :json number docs_written:  Number of documents written to target
+    :json number bulk_get_attempts: The count of attempted doc fetches
+        with ``_bulk_get``.
+    :json number bulk_get_docs: The count of successful docs read with
+        ``_bulk_get``.
```

Comment:
It's interesting (not a problem, necessiarly) that we report the number of successes here, but the number of failures for other metrics.

Reply:
The success here mirrors the `docs_reads`. I debated using a `failure` count, however `_bulk_get` is also avoided for revisions with attachments so `failure` didn't seem like it quite fit.